lms/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -626,115 +613,3 @@ def test_externalauth_login_required_course_context(self):
         self.assertEqual(shib_response.redirect_chain[-2],
                          (target_url_shib, 302))
         self.assertEqual(shib_response.status_code, 200)
-
-
-@httpretty.activate
-class LoginOAuthTokenMixin(ThirdPartyOAuthTestMixin):
```

Comment:
You meant to just delete all these tests and not move them anywhere?  I think this applies to all deleted tests through the rest of this file.

Reply:
I removed the unused `login_oauth_token` view.

Reply:
Great.  Thanks.